samples/net/syslog_net/README.rst

```diff
@@ -0,0 +1,33 @@
+.. _syslog-net-sample:
+
+Syslog net Application
+######################
+
+Overview
+********
+
+This sample application enables remote syslog service. You need to
+set the syslog server address in prj.conf. When run, the application
+will send syslog messages to remote server.
+See https://tools.ietf.org/html/rfc5424 and https://tools.ietf.org/html/rfc5424
+for more details about syslog protocol over UDP.
+
+The source code for this sample application can be found at:
+:file:`samples/net/syslog_net`.
+
+Requirements
+************
+
+- :ref:`networking_with_qemu`
+
```

Comment:
Looks like you could use some information here about the options you need to configure in ``prj.conf`` to specifically make remote syslog work.  Is it just the lines in ``prj.conf`` after the comment ``# syslog net backend config`` ?